content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -3,135 +3,180 @@ title: Instrumentation
-  // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
-}
+```shell
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-node
 ```
 
-## Initializing a Tracer
+It's recommended to have a separate `tracing.js` file that has all the SDK
+initialization code in it:
+
+```js
+import opentelemetry from "@opentelemetry/api";
+import nodeOTel from "@opentelemetry/sdk-node";
+
+const sdk = new nodeOTel.NodeSDK({
+  traceExporter: new opentelemetry.tracing.ConsoleSpanExporter(),
+});
+
+sdk.start()
+
+// Now you can create a tracer. Export it for the rest
+// of the app to use.
+export const tracer = opentelemetry.trace.getTracer(
+  'name-of-tracer-goes-here'
+);
+```
+
+Next, ensure that `tracing.js` is required in your node invocation. For example:
+
+```
+node --require './tracing.js' <other-app-file.js>
+```
 
-As you have learned in the previous [Getting Started](../getting-started) guide you
-need a TracerProvider and an Exporter. Install the dependencies and add them to
-the head of your application code to get started:
+Although it's not technically required to `--require` the tracing initialization
+for only manual instrumentation, it is necessary when you use automatic
+instrumentation. So in practice, it's still likely to be necessary.
+
+### Browser
+
+First, ensure you've got the right packages:
 
 ```shell
-npm install @opentelemetry/api
-npm install @opentelemetry/sdk-trace-base
+npm install \
+  @opentelemetry/api \
+  @opentelemetry/sdk-trace-web
 ```
 
-Next, initialize a tracer, preferably in a separate file (e.g., `instrumentation-setup.js`):
+Create a `tracing.js` file that initialized the Web SDK, creates a
+`TracerProvider`, and exports a `Tracer`.
 
 ```javascript
-const {
-  BasicTracerProvider,
-  ConsoleSpanExporter,
-  SimpleSpanProcessor,
-} = require('@opentelemetry/sdk-trace-base');
-const opentelemetry = require('@opentelemetry/api');
+import opentelemetry from "@opentelemetry/api";
+import { WebTracerProvider } from '@opentelemetry/sdk-trace-web';
 
-const provider = new BasicTracerProvider();
+const provider = new WebTracerProvider();
 
-// Configure span processor to send spans to the exporter
-provider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
-provider.register();
+// Optionally register automatic instrumentation libraries
+registerInstrumentations({
+  instrumentations: [],
+});
 
-// This is what we'll access in all instrumentation code
 export const tracer = opentelemetry.trace.getTracer(
-  'example-basic-tracer-node'
+  'name-of-tracer-goes-here'
 );
 ```
 
-This registers a tracer provider with the OpenTelemetry API as the global tracer provider, and exports a tracer instance that you can use to create spans.
-
-If you do not register a global tracer provider, any instrumentation calls will be a no-op, so this is important to do!
+You'll need to bundle this file with your web application to be able to use
```

Comment:
> This registers a global `TracerProvider` with the OpenTelemetry API, which may be used to create spans.
If no global `TracerProvider` is registered, instrumentations which use the API will use a no-op tracer and no spans will be created or exported.

I'll work out a way to incorporate this.